### ISSUE TITLE
Changes of public keys usage in tests

### ISSUE THREAD

**Issue by rwrzesien** [golemfactory/concent#43]
Here is the list of changes of public keys in tests coming from work on first use case that looks suspicious:

8adc0b4057a32614af6db02dc6e4e989d3cbf3e5 - oldest one:

https://github.com/golemfactory/concent/blob/8adc0b4057a32614af6db02dc6e4e989d3cbf3e5/concent_api/core/tests/test_core_views.py#L144
https://github.com/golemfactory/concent/blob/8adc0b4057a32614af6db02dc6e4e989d3cbf3e5/concent_api/core/tests/test_core_views.py#L148
https://github.com/golemfactory/concent/blob/8adc0b4057a32614af6db02dc6e4e989d3cbf3e5/concent_api/core/tests/test_core_views.py#L163
https://github.com/golemfactory/concent/blob/8adc0b4057a32614af6db02dc6e4e989d3cbf3e5/concent_api/core/tests/test_core_views.py#L167
https://github.com/golemfactory/concent/blob/8adc0b4057a32614af6db02dc6e4e989d3cbf3e5/concent_api/core/tests/test_core_views.py#L179
https://github.com/golemfactory/concent/blob/8adc0b4057a32614af6db02dc6e4e989d3cbf3e5/concent_api/core/tests/test_core_views.py#L183
https://github.com/golemfactory/concent/blob/8adc0b4057a32614af6db02dc6e4e989d3cbf3e5/concent_api/core/tests/test_core_views.py#L195
https://github.com/golemfactory/concent/blob/8adc0b4057a32614af6db02dc6e4e989d3cbf3e5/concent_api/core/tests/test_core_views.py#L199
https://github.com/golemfactory/concent/blob/8adc0b4057a32614af6db02dc6e4e989d3cbf3e5/concent_api/core/tests/test_core_views.py#L319
https://github.com/golemfactory/concent/blob/8adc0b4057a32614af6db02dc6e4e989d3cbf3e5/concent_api/core/tests/test_core_views.py#L393

This should PROVIDER_PUBLIC_KEY, as provider is sending ForceReportComputedTask.

https://github.com/golemfactory/concent/blob/8adc0b4057a32614af6db02dc6e4e989d3cbf3e5/concent_api/core/tests/test_core_views.py#L247
https://github.com/golemfactory/concent/blob/8adc0b4057a32614af6db02dc6e4e989d3cbf3e5/concent_api/core/tests/test_core_views.py#L251

This should REQUESTOR_PUBLIC_KEY, as requestor is sending AckReportComputedTask.

**Comment by cameel** [comment]
Yeah. So basically looks like all the key changes in all commits are wrong.

I have added numbers in the description to make referring to these points easier.

> 8adc0b4057a32614af6db02dc6e4e989d3cbf3e5 - oldest one:

So every change in this commit is wrong. The whole commit should be reverted with `git revert` and changes redone.

> This should be PROVIDER_PUBLIC_KEY, as described in tests steps. I have some issues with this one on identification by key branch, as it has impact on the logic, but I have left it as it is for now.

Here remember that the key is wrong in two places (you listed only one):
https://github.com/golemfactory/concent/blob/70c6652541d969afd20e4f635812b7fb8c56f979/concent_api/core/tests/test_integration_report_computed_task.py#L481-L491

This is basically a single change split into two commits: 442901761b72b26bc4788af062dd46c093c0c9d0 and 70c6652541d969afd20e4f635812b7fb8c56f979. You should just revert them with `git revert`.
